app/views/granules/download.sh.erb

```diff
@@ -1,20 +1,88 @@
 #!/bin/sh
 
+<% if @urls.first.start_with? 'https://datapool.asf.alaska.edu' %>
+set_cookie ()
+{
```

Comment:
Normally, you can do ```curl -c cookie -b cookie``` with ```-d password&username``` to simulate the session. However, in this case, the generated cookie file is empty. I had to generate the cookie from response manually.